.gitattributes

```diff
@@ -1 +1,2 @@
 docs/gallery/data/* filter=lfs diff=lfs merge=lfs -text
+tests/data/feature_vector/features_T301.csv filter=lfs diff=lfs merge=lfs -text
```

Comment:
just this csv?

Reply:
all tests/data/*
plus this csv, I add it for test feature collection after put all stuffs under tests/data to git-lfs

Reply:
using git lfs recommit again to ensure those are on track

Reply:
merged